assets/js/components/Select/Select.jsx

```diff
@@ -4,37 +4,45 @@ import { Listbox, Transition } from '@headlessui/react';
 import { CheckIcon, ChevronUpDownIcon } from '@heroicons/react/20/solid';
 
 import classNames from 'classnames';
+import { find, get } from 'lodash';
+import { createOptionRenderer } from '.';
 
 export const OPTION_ALL = 'all';
 
 const defaultOnChange = () => {};
-const defaultRenderOption = (item) => item;
+const defaultRenderOption = (optionsName) =>
+  createOptionRenderer(`All ${optionsName}`, (item) => item);
```

Comment:
We already discussed multiple times about not encapsulating this behavior by default into the component and I see strong pushback from you about this.

This `all` handling is a peculiarity of a single use case, and we will for sure use this Select component in other places like the tables where we currently have duplicated code or other filters like the one in the checks results page.

At this point I can just disagree and commit.

Reply:
I was doubtful myself :smile: 
I am ok with changing, it meaning that if we provide a list of options containing also the `all` option, it would be rendered just as a plain `all` text. :+1: